docker/cortx-build/Makefile

```diff
@@ -38,7 +43,7 @@ build: io-path control-path cortx-prereq release_build
 cortx-all-image: cortx-motr cortx-s3server cortx-hare cortx-prvsnr cortx-manager release_build
 
 ## cortx-all-image: generate packages required for cortx-all image including third-party-deps at "/var/artifacts/0"
-cortx-all-rockylinux-image: cortx-motr cortx-hare cortx-prvsnr cortx-manager cortx-rgw cortx-ha release_build
+cortx-all-rockylinux-image: validation cortx-motr cortx-hare cortx-prvsnr cortx-manager cortx-rgw cortx-ha release_build
```

Comment:
why `validation` only for rockylinux-image? why not other rgw targets?

Reply:
all deployment images are building at cortx-all-rockylinux-image step only in docker container hence, added validation there only. Not sure what other targets needs disk validation.

Reply:
Added at L#64 for cortx-rgw also